src/app/modular/application/types.ts

```diff
@@ -0,0 +1,23 @@
+import { User } from '../auth/types';
+
+export interface Application {
+  applicant: User,
+  id: number,
+  postId: number,
+  status: number,
+  appliedAt: Date,
+  gitHubURL: string,
+  linkedInURL: string,
+  portfolioURL: string,
+  otherURL: string,
+  resumeURL: string,
+  additionalInfo: string,
```

Comment:
```suggestion
  gitHubURL?: string,
  linkedInURL?: string,
  portfolioURL?: string,
  otherURL?: string,
  resumeURL?: string,
  additionalInfo?: string,
```

These should be marked as optional, since we'll likely use this interface to validate when people actually apply.